pkg/files/stdin.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2022 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package files
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+)
+
+var hasStdinBeenRead bool
+
+//ReadStdin only read stdin once
```

Comment:
nit: typically we leave a space between the comment token(s) and the start of the comment.